packages/esm-patient-attachments-app/src/utils.ts

```diff
@@ -28,7 +28,7 @@ export function createGalleryEntry(data: AttachmentResponse): Attachment {
   return {
     id: data.uuid,
     src: `${window.openmrsBase}${attachmentUrl}/${data.uuid}/bytes`,
-    filename: data.filename,
+    filename: data.filename.replace(/\.[^/.]+$/, ''),
```

Comment:
Since createGallary Function passes the uploaded file details to attachment which is used by the UI components ie the attachments-overview-component, and the grid and table components. These components all use the filename from the Attachment type and filename comes with the extension.

Reply:
Be nice to have a comment explaining things.